README.md

```diff
@@ -84,6 +84,14 @@ This script may also be used for updates. To update a script
 generated by the Tracim installation, update the source code with git pull and
 rerun the same script to update the database model, the system and Python dependencies.
 
+Note: In case you do not want or can not install (for example, if you are using arm64) some preview based dependencies,
+ like libreoffice, inkscape, ffmpeg, vtk, etc... You can install with the minimal preview dependencies requirements,
+ which mean less working preview but also less problem in specific environnement:
```

Comment:
```suggestion
Note: In case you do not want or can not install (for example, if you are using arm64) some dependencies for previews,
 like LibreOffice, Inkscape, FFmpeg, VTK, etc... You can install with the minimal requirements for previews,
 which means fewer working previews but also fewer problems in specific environments:
```